src/main/scala/org/deepdive/inference/Sampler.scala

```diff
@@ -79,7 +79,6 @@ class Sampler extends Actor with ActorLogging {
           "-w", weightsFile,
           "-v", variablesFile,
           "-f", factorsFile,
-          "-e", edgesFile,
```

Comment:
We should probably update the sampler documentation as well.  Could you open an issue for that so we don't forget?